pykafka/balancedconsumer.py

```diff
@@ -504,18 +515,24 @@ def _check_held_partitions(self):
             self._rebalance()
 
     def _brokers_changed(self, brokers):
+        if not self._running:
+            return False  # disables the watcher
```

Comment:
Yes, do you think I should also add `return True` at the end? It should work fine without that, because the caller distinguishes between `None` and `False`, but maybe that makes it look more even?